ext/pdo/pdo_stmt.stub.php

```diff
@@ -4,8 +4,7 @@
 
 class PDOStatement implements IteratorAggregate
 {
-    /** @var string|null */
-    public $queryString;
+    public ?string $queryString;
```

Comment:
Okay, I have looked into this in a bit more detail, and this property should not be nullable after all. While we allow writing `new PDOStatement`, this will result in a broken object, and it's okay if it has an uninitialized `$queryString`. The reason why we can't simply throw on construction as we would usually do is that it's possible to extend `PDOStatement` and specify a constructor, which will be respected by PDO. But even in that case, directly creating the object is not valid, it always needs to be created through PDO to be fully initialized.